### DISCUSSION TITLE
Welcome to bdfd-wiki Discussions!

### DISCUSSION THREAD

**Discussion by noituri** [NilPointer-Software/bdfd-wiki#115] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by R3CKhi** [comment]
hi

**Comment by blender-dev** [comment]
Hello

**Comment by CoconutNeko** [comment]


**Reply by priyanujgogoi-28** [reply]
This should solve your issue:
```
$nomention
$username[$mentioned[1]]#$discriminator[$mentioned[1]]
```

For more reference, checkout the [$username](https://nilpointer-software.github.io/bdfd-wiki/nightly/bdscript/username.html#usage-2) wiki.

---
If you have any more questions/need help, feel free to join our [Support Server](https://botdesignerdiscord.com/discord).

**Reply by priyanujgogoi-28** [reply]
You received the error because you haven't mentioned a user. Mentioning a user should fix the issue.

![Screenshot_20221218_180256](https://user-images.githubusercontent.com/95774950/208298697-14ca3a19-69f0-4c38-b90d-495ee3f0a8b6.png)

Now, let's suppose...you don't want the error to pop or want a custom error.
Then, use [`$onlyIf[]`](https://nilpointer-software.github.io/bdfd-wiki/nightly/bdscript/onlyIf.html) function with a condition to check if the author has mentioned a user.

Here's an example,
```
$nomention
$onlyIf[$mentioned[1]!=;You need to mention an user!]
$username[$mentioned[1]]#$discriminator[$mentioned[1]]
```

![Screenshot_20221218_180413](https://user-images.githubusercontent.com/95774950/208298783-b4d80bb0-b205-431d-8481-ba83f277252e.png)
